en/config_rover/ackermann.md

```diff
@@ -1,15 +1,4 @@
-# Ackermann Rover
-
-<Badge type="tip" text="main (PX4 v1.16+)" /> <Badge type="warning" text="Experimental" />
-
-An _Ackermann rover_ controls its direction by pointing the front wheels in the direction of travel — the [Ackermann steering geometry](https://en.wikipedia.org/wiki/Ackermann_steering_geometry) compensates for the fact that wheels on the inside and outside of the turn move at different rates.
-This kind of steering is used on most commercial vehicles, including cars, trucks etc.
-
-![Axial Trail Honcho](../../assets/airframes/rover/rover_ackermann/axial_trail_honcho.png)
-
-::: info
-PX4 does not require that the vehicle uses the Ackermann geometry and will work with any front-steering rover.
-:::
+# Configuration/Tuning
```

Comment:
YOu need to include Ackermann in the main page title. This is needed because it is a key search term and provides "this" context. Further, in mobile view you don't get the sidebar, so again you have lost context. This would be partially fixed with a breadcrumb, but there is no decent plugin for that yet and I'm not ready to invest the time writing one. In the sidebar I provide minimal context though, because you can see the context.

We don't need to spell out "Ackermann Rover" I don't think. The pattern so far is _`FrameType` Config Tuning_. We could use _Config/Tuning (`FrameType`)_ but this is what it is for now.

```suggestion
# Ackermann Configuration/Tuning
```